_posts/2015-04-30-gsoc-15-is-now-live.markdown

```diff
@@ -0,0 +1,122 @@
+**Mentors:** [Kir Shatrov](https://github.com/kirs) and [Carlos Antonio da Silva](https://github.com/carlosantoniodasilva)
+
+Islam is adding some of the things we should already have in Rails, liked adding named arguments
+ for Action View helpers (goodbye counting commas!). He will also be researching how to improve some of our core abstractions like
+ [ActionController::Parameters](http://edgeapi.rubyonrails.org/classes/ActionController/Parameters.html)
+ and
+ [ActionView::OutputBuffer](http://tenderlovemaking.com/2014/06/04/yagni-methods-slow-us-down.html)
+ to enable better security and performance.
+
+## Web Console Browser Extensions
+
+**Student:** [Hiroyuki Sano](https://github.com/sh19910711)    
+**Mentor**: [Genadi Samokovarov](https://github.com/gsamokovarov)
+
+Following up on the work of previous GSoC projects, Hiroyuki will be creating browser extensions for
+the Rails web console. Like the the Source Maps project, this one will give you a better live
+debugging experience using standard tools that everyone has available already.
+
+## Test Failure Prediction
+
+**Student:** [Genki Sugimoto](https://github.com/Genki-S)    
+**Mentors:** [Robin Dupret](https://github.com/robin850), **TODO** Confirm: "and maybe Josh K from travis CI"
+
+[Aaron Patterson](https://github.com/tenderlove) has touched upon some interesting ideas on
+[predicting test
+failures](http://tenderlovemaking.com/2015/02/13/predicting-test-failues.html) using the
+experimental [Coverage](http://ruby-doc.org/stdlib-trunk/libdoc/coverage/rdoc/Coverage.html) feature
+available in recent Ruby versions. Genki will be experimenting with this to see if we can make it a
+part of the Rails testing ecosystem.
+
+## Refactoring Cookies
+
+**Student:** [Siddharth Bhatore](https://github.com/sbhatore)    
+**Mentors:** [Kasper Timm Hansen](https://github.com/kaspth) and [Prem Sichanugrist](https://github.com/sikachu)
+
+Rails cookie handling is pretty basic, and although it works in most use cases, we can improve
+it. Siddarth will be adding server-side expiration mechanisms and purpose fields to our existing
+cookie jar, allowing us to have better control and security over our systems.
+
+## Improving RubyBench.org
+
+**Student:** [Kasif Gilbert](https://github.com/klgilbert)    
+**Mentors:** [Sam Saffron](https://github.com/SamSaffron)
+
+In case you're not familiar with it, [RubyBench](http://rubybench.org/) is an amazing effort to keep
+long running benchmarks for Ruby and related projects (like
+[Rails](http://rubybench.org/rails/rails/releases)). As you can see, our own benchmarks could use
+some love, so Kasif will be taking care of this. If everything goes well, JRuby support in
+RubyBench would be the next step for this project.
+
+* * *
+
+**Fun Fact**: 5 of the 13 mentors we have this year participated as GSoC students in previous years. Today they're all
+  core members or active contributors. **TODO** Confirm math, update if Josh K. is added.
```

Comment:
Emphasizing core members is a bit much when only of us is. I'd flip it to say something like: "Today they're active contributors and one is even on the Rails core team." :smile: